app/javascript/util/languageUtil.tsx

```diff
@@ -164,14 +164,18 @@ export const getSfGovUrl = (enLink: string, node?: number, path?: string) => {
  */
 export function renderMarkup(translatedString: string, allowedTags?: string) {
   return (
-    <Markdown options={{ forceBlock: true }}>
+    <Markdown options={{ forceBlock: true, namedCodesToUnicode: { "#39": "\u0027" } }}>
       {stripMostTags(translatedString, allowedTags)}
     </Markdown>
   )
 }
 
 export function renderInlineMarkup(translatedString: string, allowedTags?: string) {
```

Comment:
I grepped for the regex `renderInlineMarkup\([^t]` and found that this function is used for free text fields in some places.